src/torrent.cpp

```diff
@@ -1001,7 +1000,7 @@ namespace libtorrent
 		{
 			throw;
 		}
-		catch (system_error const& err)
+		catch (boost::system::system_error const& err)
```

Comment:
yes, but it's like it's not getting it, it just fail to compile, I will look at it